### PR TITLE
nodejs-lts: Update to version 20.11.1

### LINKED ISSUE #135: nodejs-lts: issue while verifying the version
Error from the excavator
```text
nodejs-lts: couldn't match 'LTS Version: <strong>([\d.]+)</strong>' in https://nodejs.org/en/download/
```

Investigation is required

### PR DESCRIPTION
**Pull request by alexchapar**
## What type of Apps related PR is this? (check all applicable)
- [ ] ✨ Add Manifest
- [ ] ♻️ Refactor
- [x] 🐛 Bug Fix
- [ ] 👷 Optimization
- [x] 🚩 Other
## Description
- Fix the checkver regex to find the latest version
- Add support for arm64
## Related Tickets & Documents
Fixes #135
## [optional] What gif/emoji best describes this PR or how it makes you feel?
:alien: :bug: :sparkles: :bento: 
### Add/Modify a description in the manifest?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Check URLs?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Update to the latest version?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Update Hashes?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Format the json manifest?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Notes in the manifest?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### Application Dependencies?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### Application Suggestions?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### Pre-installation steps to perform?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### Post-installation steps to perform?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### File persistence to perform?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### CODE REVIEW

**Comment by BotServiceAutomation** [comment]
All changes look good.

Wait for review from human collaborators.
### nodejs-lts

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate
- [x] Autoupdate Hash Extraction

**Review by Bibz87 — APPROVED** [review]

**Pull request merged**
